app/routes/lending/LendableObjectsList.css

```diff
@@ -0,0 +1,46 @@
+@import url('~app/styles/variables.css');
+
+.myRequestsList {
+  margin-bottom: 30px;
+}
+
+.searchBar {
+  margin-bottom: 2rem;
+}
+
+.lendableObjectsContainer {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-gap: 2rem;
+}
+
+.lendableObjectCard {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  padding: 0;
+}
+
+.lendableObjectImage {
+  height: 15rem;
+  object-fit: cover;
+}
+
+.lendableObjectFooter {
+  display: flex;
+  justify-content: center;
+  color: var(--lego-font-color);
+  background-color: var(--additive-background);
```

Comment:
Not a fan of using the additive background as a whole background. It sticks out too much from the rest of the webapp